app/javascript/components/action-button/component.spec.js

```diff
@@ -35,15 +34,44 @@ describe("<ActionButton />", () => {
   });
 
   // Check if transparent class is set
-  it.todo("renders transparent button");
+  //it.todo("renders transparent button");
+  it("renders transparent button", () => {
+    const newProps = {
+      ...props,
+      isTransparent: true
+    }; 
+    
+    mountedComponent(<ActionButton {...newProps} />, state);
+    expect(screen.getAllByRole('button', { className: 'isTransparent' })).toHaveLength(1);
+   
+  });
 
   // Check if outlined class is set
-  it.todo("renders outlined button");
+  //it.todo("renders outlined button");
+
+  //it.todo("renders a pending state");
```

Comment:
Uncomment this line, so devs will know where test need to be added

Reply:
I have uncommented these line and move these line at the end .